src/DynamoCoreWpf/Controls/StandardPanel.xaml

```diff
@@ -5,106 +5,114 @@
+                                <Trigger Property="IsFocused"
+                                         Value="True">
+                                    <Setter Property="Background"
+                                            Value="{StaticResource LibraryMemberOnHover}" />
+                                </Trigger>
+                            </ControlTemplate.Triggers>
+                        </ControlTemplate>
+                    </Setter.Value>
+                </Setter>
+                <EventSetter Event="PreviewMouseLeftButtonDown"
+                             Handler="OnMouseLeftButtonDown" />
+                <EventSetter Event="KeyDown"
+                             Handler="OnMemberButtonKeyDown" />
+                <EventSetter Event="MouseEnter"
+                             Handler="OnListBoxItemMouseEnter" />
+                <EventSetter Event="MouseLeave"
+                             Handler="OnPopupMouseLeave" />
+            </Style>
+            <Style x:Key="MoreButtonStyle"
+                   TargetType="{x:Type Button}">
+                <Setter Property="Template">
+                    <Setter.Value>
+                        <ControlTemplate>
+                            <Grid x:Name="container">
+                                <TextBlock x:Name="text"
+                                           HorizontalAlignment="Left"
+                                           VerticalAlignment="Center"
+                                           TextAlignment="Center"
+                                           Foreground="#4691CE"
+                                           FontSize="12"
+                                           Margin="71,0,0,0">
+                                    <TextBlock.Text>
+                                        <Binding Path="Content"
+                                                 RelativeSource="{RelativeSource TemplatedParent}" />
+                                    </TextBlock.Text>
+                                </TextBlock>
+                            </Grid>
+                            <ControlTemplate.Triggers>
+                                <Trigger Property="Button.IsMouseOver"
+                                         Value="true">
+                                    <Setter TargetName="container"
+                                            Property="Background"
+                                            Value="#2F2F2F" />
+                                </Trigger>
+                            </ControlTemplate.Triggers>
+                        </ControlTemplate>
+                    </Setter.Value>
+                </Setter>
+            </Style>
+
+        </ResourceDictionary>
     </UserControl.Resources>
     <UserControl.Visibility>
         <Binding Path="ClassDetailsVisibility"
-                 Converter="{StaticResource BoolToVisibilityCollapsedConverter}" />
+                 Converter="{StaticResource BooleanToVisibilityCollapsedConverter}" />
```

Comment:
Is `BooleanToVisibilityCollapsedConverter` defined? I only saw `BoolToVisibilityCollapsedConverter` in the latest  code.